checks.sh

```diff
@@ -1,48 +1,110 @@
 #! /bin/sh
+# arguments:
+# 1. a relative path to a directory
+# 2. flag to flip exit codes for success and failure (used in testing)
+# 3. flag to ignore whitespace when diffing
 
```

Comment:
What if no arguments is given?

You could check it like:
```shell
if [ "$#" -gt 0 ]; then
...
```

Reply:
- if the 1st arg is left unassigned, we don't ``cd`` into another directory
- if the 2nd arg is left unassigned, FAILURE_EXPECTED is set to '0'
- if the 3rd arg is left unassigned, DIFF_IGNORE_WHITESPACE is set to '1'